Dockerfile

```diff
@@ -1,16 +1,18 @@
 ARG GOVERSION=1.17
 FROM golang:${GOVERSION} as builder
+USER 65534
 ARG GOARCH
 ENV GOARCH=${GOARCH}
+ENV GOCACHE=/tmp/.cache
 WORKDIR /go/src/k8s.io/kube-state-metrics/
-COPY . /go/src/k8s.io/kube-state-metrics/
+COPY --chown=65534:65534 . /go/src/k8s.io/kube-state-metrics/
 
 RUN make build-local
 
 FROM gcr.io/distroless/static:latest
-COPY --from=builder /go/src/k8s.io/kube-state-metrics/kube-state-metrics /
+USER 65534
 
-USER nobody
+COPY --from=builder --chown=65543:65534 /go/src/k8s.io/kube-state-metrics/output/kube-state-metrics /
```

Comment:
why does the UID differ from the GID? Shouldn't it be the same?

Reply:
Yes, you're right. Was a typo on my side. Fixed it few seconds ago - sry ;-)